docker-compose.yml

```diff
@@ -9,6 +9,24 @@ services:
                 pipenv_dev_arg: ${PIPENV_DEV_ARG}
                 python_version: ${PYTHON_VERSION}
         image: elifesciences/profiles_venv:${IMAGE_TAG}
+    cli:
+        build:
+            context: .
+            dockerfile: Dockerfile
+            args:
+                image_tag: "${IMAGE_TAG}"
+                python_version: ${PYTHON_VERSION}
+        image: elifesciences/profiles:${IMAGE_TAG}
+        command: venv/bin/python
+        volumes:
+            - ./config/app.cfg:/srv/profiles/app.cfg
+            - ./config/clients.yaml:/srv/profiles/clients.yaml
+            - ./config/uwsgi.ini:/srv/profiles/uwsgi.ini
```

Comment:
Needed?

Reply:
I think `uwsgi.ini` can be removed because it's passed to a process that is not executed in this container. Instead `clients.yaml` is loaded by `manage.py` which is possible to run with the `cli` container, hence should be provided to support `docker-compose run cli manage.py something`.